### DISCUSSION TITLE
Больше данных?

### DISCUSSION THREAD

**Discussion by Psytoshgen** [IATkachenko/HA-YandexWeather#22] [General]
Салют, до этого юзал Гисметео, плюс GDACS (или как он там).
Есть ли возможность добавить показания Геомагнитного поля, УФ индекс и (если можно) погодные предупреждения? Жена очень зависима от геомагнитки... 
Во стальном - все отлично, но не ясно почему большая часть показаний отключена изначально интеграцией... 

И я там не дочитал, дочитаю, почему всего 2 прогноза... хотелось бы на каждый час или 3 часа... сейчас дочитаю. 

Если тут гуляют Господа из Яндекса, Парни, откройте показатели, это не коммерческий проект, все мы тут от вас так или иначе зависим. 
Всем добра!

**Comment by IATkachenko** [comment]
Здравствуйте.

Геомагнитное поле, УФ индекс и предупреждения Яндекс не отдает вообще. Так что увы, никак.

Сенсоры отключены изначально просто потому что все это есть в погодном (weather) компоненте и карточки погоды для UI и без дополнительных сенсоров показывают все полезные данные. А то что в карточки погоды не входит -- для этого сенсоры по умолчанию активированы.

2 шести часовых интервала прогноза -- опять же, столько Яндекс отдает. За 30к/месяц отдает больше, но нашей интеграцией платынй тариф не поддерживается (и крайне мало вероятно что будет).
